app/AppKernel.php

```diff
@@ -14,7 +14,6 @@ public function registerBundles()
             new Symfony\Bundle\TwigBundle\TwigBundle(),
             new Symfony\Bundle\MonologBundle\MonologBundle(),
             new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
-            new Symfony\Bundle\AsseticBundle\AsseticBundle(),
```

Comment:
assetic is removed from core SE. I think the only place we require assetic is with the CreateBundle, which isn't used in the standard edition.

Maybe it's a better idea to remove the Assetic dep in CreateBundle.